### ISSUE TITLE
Please provide better build instructions

### ISSUE THREAD

**Issue by martinvonwittich** [jkcoxson/netmuxd#35]
While attempting to build `netmuxd` with `cargo build`, I stumbled over both #31 and #23. The current instruction:

> Make sure both [rusty_libimobiledevice](https://github.com/jkcoxson/rusty_libimobiledevice) and [plist_plus](https://github.com/jkcoxson/plist_plus) are buildable.

wasn't especially useful, because the failure to build `plist_plus` was the first error I encountered and therefore tried to fix, but [plist_plus](https://github.com/jkcoxson/plist_plus) doesn't actually provide any instructions.

The required steps to fix the build on my Ubuntu 22.04 machine were:

1. Install the libimobiledevice build dependencies as explained in https://github.com/jkcoxson/netmuxd/issues/23#issuecomment-1283046960 (though I skipped `libimobiledevice-glue-dev` which is apparently not yet available in Ubuntu 22.04, but also not required yet)
2. Build with `cargo build --bin netmuxd` instead of `cargo build` as explained in https://github.com/jkcoxson/netmuxd/issues/31#issuecomment-1559051762

**Comment by jkcoxson** [comment]
This crate has been simplified significantly, does this solve your problem?

**Comment by martinvonwittich** [comment]
> This crate has been simplified significantly, does this solve your problem?

Yes, the new build instruction `cargo build --release` runs fine now.
